### PR TITLE
Ack canceled tasks

### PR DESCRIPTION
**Pull request by lferran**

### CODE REVIEW

File: guillotina_amqp/worker.py [hunk 1]
```diff
@@ -132,13 +131,15 @@ def num_running(self):
...
+            # Ack so that canceled job is removed from main queue
+            await channel.basic_client_ack(delivery_tag=envelope.delivery_tag)
+            return
 
         try:
             await ts.acquire()
         except TaskAlreadyAcquired:
             logger.warning(f"Task {_id} is already running in another worker")
-            # TODO: ACK task
+            return
```
Comment (vangheem) [inline guillotina_amqp/worker.py]: We probably should ack here as well.

We always need to ack or nack a message.

If you just drop out and don't do anything with the message, the message stays unacked in the queue until that worker's amqp connection is killed. Then it goes back into the queue again.

**Comment by vangheem** [comment]
5.0.21 released

**Comment by lferran** [comment]
@vangheem I think it should have been 5.0.20

**Comment by vangheem** [comment]
you're right! I was. I copied the wrong version string. :)

**Pull request merged**